hat/app/org/hatdex/hat/api/service/richData/RichDataService.scala

```diff
@@ -257,6 +256,23 @@ class RichDataService @Inject() (implicit ec: DalExecutionContext) {
     }
   }
 
+  def getRecord(
```

Comment:
we need a couple of test cases added in `RichDataServiceSpec`

Reply:
Yep

Reply:
FYI this has been deprioritised in favour of documentation and im unsure whether it will be complete by me leaving thus this PR